docs/core/notice/mobile-work-plan.md

```
@@ -0,0 +1,634 @@
+- 라우트: 문자열 상수만 정의
+
+### Server API 완료 후 Mobile 작업 (순차 진행)
+
+❌ **Group 2~5**:
+- API Service: 실제 서버 엔드포인트 호출 필요
+- Controller: API Service 의존
+- View: Controller 의존
+
+### 실행 순서 (Server ↔ Mobile 동기화)
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)